concepts/sepsis/sepsis3.sql

```diff
@@ -0,0 +1,57 @@
+WITH s1 as 
+(
+  SELECT 
+    sofa.* 
+    , soi.subject_id
+    , soi.ab_id
+    , soi.antibiotic
+    , soi.antibiotic_time
+    , soi.culture_time
+    , soi.suspected_infection
+    , soi.suspected_infection_time
+    , soi.specimen
+    , soi.positive_culture
+  FROM `lcp-internal.sepsis_test.suspicion_of_infection` as soi
+  INNER JOIN `lcp-internal.sepsis_test.sofa` as sofa
+    ON soi.stay_id = sofa.stay_id 
+    AND sofa.starttime >= DATETIME_SUB(soi.suspected_infection_time, INTERVAL 48 HOUR)
+    AND sofa.endtime <= DATETIME_ADD(soi.suspected_infection_time, INTERVAL 24 HOUR)
+  WHERE sofa.stay_id is not null and soi.stay_id is not null
+)
+, s2 as 
+(
+  SELECT distinct 
```

Comment:
ack! I don't like distinct, feels like then you did a join wrong :) i'd prefer some kind of ROW_NUMBER() logic to select the earliest suspected infection time for each successfully joined row, etc.